specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -139,6 +139,29 @@ int main()
 }
 ```
 
+## Registering for App Notifications using assets
+
+For Unpackaged applications, the developer can opt to call into Register API with DisplayName and Icon.
+WinAppSDK will register the application and display these assets when an App Notification is received.
+The developer should provide both the assets or not provide them at all. If assets are not provided WinAppSDK
+will determine appropriate DisplayName and Icon. Icon provided by the developer should be a valid supported
+format like png, bmp, ico or jpg and should reside on the local machine only. For Packaged applications,
```

Comment:
This seems very vague. We should probably support a handful of formats and being very explicit in what those are, throwing an error when not valid.